lms/templates/partials/analytics.html

```diff
@@ -46,25 +17,73 @@
   else:
     courseName = 'Not in course'
   endif
-
-  appsemblerLmsTrackingCode = get_global_settings().get('appsembler_lms_tracking_code')
 %>
 
-<script>
-  % if get_global_settings().get('appsembler_lms_tracking_code'):
-    ga('create', '${appsemblerLmsTrackingCode}', 'auto', 'appsemblerLmsTracker');
-    ga('appsemblerLmsTracker.send', 'pageview', {
-      %if USER_EMAIL_HASH:
-        'userId': '${USER_EMAIL_HASH}',
-      %endif
-      'dimension1': '${siteOrganization}',
-      'dimension2': '${userType}',
-      'dimension3': '${courseName}',
-      'dimension4': 'LMS'
-    });
-  % endif
-</script>
-<!-- /new analytics -->
+% if (get_global_settings().get('customer_gtm_enabled') and (get_global_settings().get('customer_gtm_id') != "")):
+  <%
+    customerGtmId = get_global_settings().get('customer_gtm_id')
+  %>
+  <script>
+    dataLayer = [{
+      'visitorType': '${userType}',
+      'courseName': '${courseName}'
+    }];
+  </script>
+  <!-- Google Tag Manager -->
+  <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+  new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+  j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+  'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+  })(window,document,'script','dataLayer','${customerGtmId}');</script>
+  <!-- End Google Tag Manager -->
+% endif
+
+% if SHOW_GOOGLE_ANALYTICS:
+  <!-- Google Analytics -->
+  <script>
+    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+    })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+    ga('create', '${GOOGLE_ANALYTICS_APP_ID}', 'auto');
+    %if USER_EMAIL_HASH:
+      ga('set', 'userId', '${USER_EMAIL_HASH}');
+      ga('set', 'dimension1', '${USER_EMAIL_HASH }');
+    %endif
+    ga('send', 'pageview');
+  </script>
+  <!-- End Google Analytics -->
+
+  <!-- Add new Analytics system. Note: for now we'll be duplicating analytics while we're testing the new Analytics organisation. Once we're satisfied with how it works, we'll remove the old one and keep just the new one. -->
+  <!--
+    usedId: we keep a consistent userId in Analytics in order to keep tracking a single user properly
+    Dimension1: Organisation
+    Dimension2: LMS Visitor type
+    Dimension3: Visited course name
+    Dimension4: LMS or Studio
+  -->
+
+  <%
+    appsemblerLmsTrackingCode = get_global_settings().get('appsembler_lms_tracking_code')
+  %>
+
+  <script>
+    % if get_global_settings().get('appsembler_lms_tracking_code'):
+      ga('create', '${appsemblerLmsTrackingCode}', 'auto', 'appsemblerLmsTracker');
```

Comment:
```suggestion
      ga('create', '${appsemblerLmsTrackingCode | n, js_escaped_string}', 'auto', 'appsemblerLmsTracker');
```

It won't hurt if we also added it to our code.